app/initializers/extensions.js

```diff
@@ -1,4 +1,4 @@
-import '../extensions/ember-table/component';
+import 'open-event-frontend/extensions/ember-table/component';
```

Comment:
Are you sure there should not be `initialisers` before the extension in here @maze-runnar ?